src/main/java/com/hierynomus/smbj/share/Share.java

```diff
@@ -40,6 +40,8 @@
 
 public class Share implements AutoCloseable {
     private static final Logger logger = LoggerFactory.getLogger(Share.class);
+    private static final EnumSet<NtStatus> SUCCESS = EnumSet.of(NtStatus.STATUS_SUCCESS);
```

Comment:
😸 I changed a lot of fields and parameters from EnumSet to Set for added flexibility (for instance to allow using Guava ImmutableSet). Doesn't matter as much here since this is an internal constant. However you prefer.